include/openrave/kinbody.h

```diff
@@ -142,6 +142,9 @@ class OPENRAVE_API JointControlInfo_RobotController : public InfoBase
     std::string controllerType; ///< the type of the controller used to control this joint
     //int robotId = -1; ///< index of the robot within that controller?
     boost::array<int16_t, 3> robotControllerAxisIndex = {-1, -1, -1}; ///< indicates which DOF in the robot controller controls which joint axis (up to the DOF of the joint). -1 if not specified/not valid.
+    boost::array<dReal, 3> robotControllerAxisMult = {1.0, 1.0, 1.0}; ///< indicates the multiplier to convert the joint values from the environment units to the robot controller units, per joint axis. (valueInRobotControllerUnits - robotControllerAxisOffset) / robotControllerAxisMult = valueInEnvUnits
```

Comment:
@rdiankov Could you check this comment? 

Current definition is, joint value in orenv unit (usually rad or m) is given by (raw_pulse - ``robotControllerAxisOffset``) / ``robotControllerAxisMult``

Even though this is just a comment and openrave doesn't use ``robotControllerAxisMult`` and ``robotControllerAxisOffset``, our applications is to follow this definition, so quite important.